test/features/support/hooks.js

```diff
@@ -0,0 +1,96 @@
+// Before And After hooks used while feature executes
```

Comment:
@GeorgeChatzigiannis  Would it be clearer if we assigned these directly to the global object?

```
 global.signIn = require('../page_objects/signInPage')
    global.landingPage = require('../page_objects/landingPage')
    global.signInFailure = require('../page_objects/signInFailure')
    global.managePupilPage = require('../page_objects/managePupilPage')
    global.helpers = require('../lib/helpers')
    global.mongo = require('../lib/mongoDbHelper')
```

@shekss - is it possible to avoid all these global dependencies?